### DISCUSSION TITLE
Programming Help , JavaScript

### DISCUSSION THREAD

**Discussion by winters-rose20** [community/community#21490] [Other Feature Feedback, Questions, & Ideas]
<p>Good morning guys.<br>
I have just began my programming journey and I would like to know if i need to learn the older version of JavaScript before I learn ES6, or can I learn ES6 without any prior knowledge to JavaScript?</p>

**Comment by ghost** [comment, marked as answer]
<p>While ES6 and vanilla JS are quite same (except ES6 has more features such as classes or arrow functions), but you should learn vanilla javascript first.</p>

**Comment by winters-rose20** [comment]
<p>Thank you BayoDino :hugs:</p>

**Comment by winters-rose20** [comment]
<p>Thank you for your response​:hugs::hugs:</p>
